### PR TITLE
Rough script for generating neuroglancer segmentation layers.

### PR DESCRIPTION
**Pull request by perlman**
Here is a script version of the code used to convert @ogliko's segmentation to a neuroglancer precomputed volume.

I still have a few things to do before we merge this in:
- [x] Use argschema (like the other scripts in this repo)
- [x] Fix default voxel sizes

### CODE REVIEW

File: acpreprocessing/neuroglancer/segmentation.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+       out_path: directory with the previously generated segmentation layer.
+       
+    """
+    # There a few cloud-volume bugs we are working around:
+    #   * The info file is not generated
+
+    vol = CloudVolume(f'file://{out_path}', compress='')
+    files = glob.glob(f"{source_path}/swc_files_nm/*.swc")
+    files = sorted(files)
+    skel_dir = os.path.join(out_path, "skeletons")
```
Comment (RussTorres) [inline acpreprocessing/neuroglancer/segmentation.py:89]: Is there a complication in using `vol.skeleton.upload` or `vol.skeleton.upload_raw`?  If not, this should make e.g. sharded support easier too.

File: acpreprocessing/neuroglancer/segmentation.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+import numpy as np
+import tifffile
+import glob
+import os
+import json
+
+import argschema
+import argschema.validate
+
+from cloudvolume import CloudVolume, Skeleton
```
Comment (RussTorres) [inline acpreprocessing/neuroglancer/segmentation.py:27]: Let's add cloud-volume to requirements.txt

**Review by RussTorres — CHANGES_REQUESTED** [review]
Requesting small changes, but I am all for integrating with cloudvolume for this and other things.  To that end, I'm suggesting using some more of the built-in functionality if possible and adding it to the repo requirements.

**Review by RussTorres — APPROVED** [review]

File: acpreprocessing/neuroglancer/segmentation.py (hunk 1 above)
Comment (perlman) [reply acpreprocessing/neuroglancer/segmentation.py:89]: I will address this after submitting a small PR to cloud-volume to allow uncompressed skeleton generation.

**Pull request merged**